docs/reference/filebeat/processor-translate-guid.md

```diff
@@ -35,18 +35,40 @@ The `translate_ldap_attribute` processor has the following configuration setting
 | --- | --- | --- | --- |
 | `field` | yes |  | Source field containing a GUID. |
 | `target_field` | no |  | Target field for the mapped attribute value. If not set it will be replaced in place. |
-| `ldap_address` | yes |  | LDAP server address. eg: `ldap://ds.example.com:389` |
-| `ldap_base_dn` | yes |  | LDAP base DN. eg: `dc=example,dc=com` |
-| `ldap_bind_user` | no |  | LDAP user. |
-| `ldap_bind_password` | no |  | LDAP password. |
+| `ldap_domain` | no |  | DNS domain name (e.g., `example.com`) used for DNS SRV discovery and to construct FQDNs from `LOGONSERVER`. When omitted Beats inspects OS metadata to infer the domain (Windows: `USERDNSDOMAIN`, `GetComputerNameEx`, TCP/IP + Kerberos registry keys, hostname; Linux/macOS: `/etc/resolv.conf`, `/etc/krb5.conf`, hostname). |
+| `ldap_address` | no |  | LDAP server address (e.g., `ldap://ds.example.com:389`). When omitted Beats auto-discovers controllers by querying `_ldaps._tcp.<domain>` first, `_ldap._tcp.<domain>` second, and finally the Windows `LOGONSERVER` variable if available. Candidates are tried in order until one succeeds. |
+| `ldap_base_dn` | no |  | LDAP base DN (e.g., `dc=example,dc=com`). When omitted Beats queries the server's rootDSE for `defaultNamingContext`/`namingContexts`. If the controller does not expose those attributes, client initialization fails and you must configure the value manually. |
+| `ldap_bind_user` | no |  | LDAP DN/UPN for simple bind. When provided with `ldap_bind_password` Beats performs a standard bind. When set without a password Beats issues an unauthenticated bind using this identity (useful for servers that expect a bind DN even for anonymous operations). |
+| `ldap_bind_password` | no |  | LDAP password for simple bind. When both the username and password are omitted Beats attempts automatic authentication: on Windows it first tries SSPI with the Beat's service or user identity using the SPN `ldap/<hostname derived from ldap_address>` and falls back to an unauthenticated bind if that fails. Non-Windows platforms immediately use an unauthenticated bind. |
 | `ldap_search_attribute` | yes | `objectGUID` | LDAP attribute to search by. |
 | `ldap_mapped_attribute` | yes | `cn` | LDAP attribute to map to. |
 | `ldap_search_time_limit` | no | 30 | LDAP search time limit in seconds. |
-| `ldap_ssl`\* | no | 30 | LDAP TLS/SSL connection settings. |
+| `ldap_ssl`* | no |  | LDAP TLS/SSL connection settings. See [SSL](/reference/filebeat/configuration-ssl.md). |
```

Comment:
Is the default for this setting changing? It also looks like the contents of the footnote was moved up into the table so we can remove the `*` in the first cell.

```suggestion
| `ldap_ssl` | no | {applies_to}`stack: ga 9.2.3` no default<br>{applies_to}`stack: ga 9.0.0` `30` | LDAP TLS/SSL connection settings. Refer to [SSL](/reference/filebeat/configuration-ssl.md). |
```